libpod/container.go

```diff
@@ -262,6 +263,12 @@ type ContainerConfig struct {
 	// File containing the conmon PID
 	ConmonPidFile string `json:"conmonPidFile,omitempty"`
 	// TODO log options for log drivers
+
+	PostConfigureNetNS bool `json:"postConfigureNetNS"`
+
+	// UserNSRoot is the directory used as root for the container when using
+	// user namespaces.
+	UserNSRoot string `json:"userNSRoot,omitempty"`
```

Comment:
Throw this up in storage config with IDMappings

Reply:
in what type do I move this information?